### PR TITLE
GK-14: Merge GK-14-modify-the-react-app-to-have-the-map-in-a-frame to main

### PR DESCRIPTION
**Pull request by ejm6xc**
Pull request created in: https://geoknow.atlassian.net/browse/GK-14 by the Git Integration for Jira app.

### CODE REVIEW

File: app/src/App.css [hunk 1]
```diff
@@ -38,6 +38,7 @@
 }
 
 .leaflet-container {
-  width: 100vw;
+  margin-left: 15vw;
```
Comment (ejm6xc) [inline app/src/App.css:41]: lol

**Review by IsaacHodg2 — APPROVED** [review]
Seems good to me

**Review by ejm6xc — COMMENTED** [review]
:)

**Review by IsaacHodg2 — APPROVED** [review]
Even better this time

**Pull request merged**